client/src/services/api.js

```diff
@@ -0,0 +1,65 @@
+// Singleton module for making preconfigured api fetch requests.
+// Also takes care of authentication management
+// abstracted to make testing easier
+import axios from 'axios';
+
+var api, isLoggedIn;
+
+// fallback baseURL string value is because of package.json's proxy entry
+function init({
+  token = window.localStorage.getItem('token'),
+  baseURL = (api && api.defaults.baseURL) || '/api'
+} = {}) {
```

Comment:
It seems that the baseUrl starts with localhost:3000 instead of localhost:3001

Reply:
Yes, the frontned starts at localhost:3000 (react's default port), and our backend uses localhost:3001 (on `index.js - line 30: const PORT = process.env.PORT || 3001`)